### DISCUSSION TITLE
GitHub Copilot doesn't let me log in

### DISCUSSION THREAD

**Discussion by GabrielComisaru** [community/community#43816] [Copilot Conversations] [answered]
### Select Topic Area

General

### Body

Trying to login into Github Copilot from VS Code and IDE Intellij and neither works even though yesterday it worked just fine. For some reason, it logged me out of it, and now every time I try to log in it brings me to the page to buy a subscription for Copilot. I've already verified myself as a student on Github Education and it tells me it doesn't need to reverify so I don't know how to fix the problem

**Comment by jtmoon79** [comment]
I have the same problem. I signed up for Github Copilot from one computer using Visual Studio Code. It worked.

When I began work on a different computer using a synced Visual Studio Code, I was unable to use GitHub Copilot.

Following these instructions https://docs.github.com/en/copilot/getting-started-with-github-copilot/getting-started-with-github-copilot-in-visual-studio-code

1. my Visual Studio Code settings are synced.
2. from Visual Studio Code, I opened the Extension _GitHub Copilot_.
3. I clicked _Subscription Required_. This opened a browser window to `https://github.com/github-copilot/signup`.
   This then forwards to the github billing page. `https://github.com/settings/billing`
   Under section _Add-ons_ I see _GitHub Copilot_ and the messages "_Your free trial expires in 59 days_".
   **I never see the pop-op dialog in Visual Studio Code to _Sign in to access GitHub Copilot_**

It seems that Github Copilot extension does not move to different instances of Visual Studio Code.

**Comment by GabrielComisaru** [comment, marked as answer]
nvm it works now, it just expires monthly and github needs to renew it and it takes "time"
